src/eca/llm_api.clj

```diff
@@ -34,6 +35,7 @@
         (string/join "\n" (subvec lines start end)))
       content)))
 
+;; NTOE: this is not going to scale ;-)
```

Comment:
yeah, I think we can easily create a util that receives the provider and do this check automagically, was waiting for more cases to understand usage